validation-tests/feature-validation.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
```

Comment:
This file should be renamed since we're going to have a lot of feature validation.